02-microarray/clustering_microarray_01_heatmap.Rmd

```diff
@@ -118,17 +118,30 @@ Your example analysis folder should now look something like this (except with re
 <img src="https://github.com/AlexsLemonade/refinebio-examples/raw/40e47f4d3f39283effbd9843a457168061be9680/template/screenshots/analysis-folder-structure.png" width=400>
 
 In order for our example here to run without a hitch, we need these files to be in these locations so we've constructed a test to check before we get started with the analysis.
-Run this chunk to double check that your files are in the right place.
+These chunks will declare your file paths and double check that your files are in the right place.
+
+First we will declare our file paths to our data and metadata files, which should be in our data directory.
+This is handy to do because if we want to [switch the dataset](#using-a-different-refinebio-dataset-with-this-analysis) we are using for this analysis, we will only have to change the file path here to get started. 
```

Comment:
Unfortunately, this link to the #using-a-different-refinebio-dataset-with-this-analysis section does not link to the section as ideally as one would hope in the rendered html file, I am thinking this is because the section we are linking to is immediately after.

The link does change to `....html#using-a-different-refinebio-dataset-with-this-analysis` but the user would not be sent to the exact location (this may be a user error on my part so you may want to try to reproduce this issue).

Perhaps we can change this everywhere to:
```suggestion
This is handy to do because if we want to [switch the dataset](#using-a-different-refinebio-dataset-with-this-analysis) we are using for this analysis as outlined in the next section, we will only have to change the file path here to get started.
```

Reply:
This is odd that this isn't working. It shouldn't matter that it's the next section. And I could see that maybe the numbering throws it off, but the numbering didn't throw off the other section link `#check-out-our-file-structure` so that feels inconsistent.

Reply:
And the only reason I don't want to use numbering in the link is that some sections like ORA and others not yet written in advanced topics will have more rearrangements of this sections so the numbering may not be identical, so I'd hate to have to alter the numbering each time (which we shouldn't have to do??).

I copied and pasted from DocToc output just to make extra sure there's no typos that are breaking it. That's not it.

Reply:
Nope, even using `#3_Using_a_different_refinebio_dataset_with_this_analysis` doesn't work even though that works normally: https://alexslemonade.github.io/refinebio-examples/02-microarray/clustering_microarray_01_heatmap.html#3_Using_a_different_refinebio_dataset_with_this_analysis

Reply:
Well, I don't know how else to fix this and it is just thee next section, so I'll resort to sans-link just telling them there will be more info about switching datasets in the next section.

Reply:
Very strange situation, but your final plan to just tell them there will be more info about switching datasets in the next section sounds good to me 👍